docs/MultiTableDatabase-batchsource.md

```diff
@@ -51,6 +51,7 @@ By default reads all tables matching the tableNamePattern.
 **blackList**: Used in conjunction with tableNamePattern, this configuration specifies the tables to be skipped.
 By default the black list is empty which means no tables will be skipped.
 
+**numSplit**: Number of splits per table. By default is 1
```

Comment:
docs should use the property names that appear in the UI. Can you update this doc to do that?

Reply:
done